Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -8514,26 +8517,26 @@
+		FA580B2B2DA64B5F005E4965 /* UnifiedSource165.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource165.cpp; sourceTree = "<group>"; };
+		FA580B2C2DA64B5F005E4965 /* UnifiedSource166.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource166.cpp; sourceTree = "<group>"; };
+		FA580B2D2DA64B5F005E4965 /* UnifiedSource167.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource167.cpp; sourceTree = "<group>"; };
+		FA580B2E2DA64B5F005E4965 /* UnifiedSource168.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource168.cpp; sourceTree = "<group>"; };
+		FA580B2F2DA64B5F005E4965 /* UnifiedSource169.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource169.cpp; sourceTree = "<group>"; };
+		FA580B302DA64B5F005E4965 /* UnifiedSource170.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource170.cpp; sourceTree = "<group>"; };
+		FA580B312DA64B5F005E4965 /* UnifiedSource171.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource171.cpp; sourceTree = "<group>"; };
+		FA580B322DA64B5F005E4965 /* UnifiedSource172.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource172.cpp; sourceTree = "<group>"; };
+		FA580B332DA64B5F005E4965 /* UnifiedSource173.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource173.cpp; sourceTree = "<group>"; };
+		FA580B342DA64B5F005E4965 /* UnifiedSource174.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource174.cpp; sourceTree = "<group>"; };
+		FA580B352DA64B5F005E4965 /* UnifiedSource175.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource175.cpp; sourceTree = "<group>"; };
+		FA580B362DA64B5F005E4965 /* UnifiedSource176.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource176.cpp; sourceTree = "<group>"; };
+		FA580B372DA64B5F005E4965 /* UnifiedSource177.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource177.cpp; sourceTree = "<group>"; };
+		FA580B382DA64B5F005E4965 /* UnifiedSource178.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource178.cpp; sourceTree = "<group>"; };
+		FA580B392DA64B5F005E4965 /* UnifiedSource179.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource179.cpp; sourceTree = "<group>"; };
+		FA580B3A2DA64B5F005E4965 /* UnifiedSource180.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = UnifiedSource180.cpp; sourceTree = "<group>"; };
```

Comment:
These got auto-added by our Xcode pre-commit hook, so I didn't have an opportunity to undo them, but it's a legitimate cleanup. Xcode will collapse `name` and `path` fields down to just one path when they match.